src/shared/ResultNotification.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Notification } from "@sailthru/stui-components";
+import styles from "./styles.module.css";
+
+function ResultNotification({ display, message, type }) {
+  if (!display) {
+    return null;
+  }
```

Comment:
If you aren't explicitly setting display, I'd suggest removing it and using the other 2 variables for this check
```suggestion
  if (!type || !message) {
    return null;
  }
```

Reply:
I've updated this to specifically set display